Std/Data/List/Lemmas.lean

```diff
@@ -451,6 +451,33 @@ theorem Sublist.eq_of_length_le (s : l₁ <+ l₂) (h : length l₂ ≤ length l
     | refl => apply Sublist.refl
     | step => simp [*, replicate, Sublist.cons]
 
+theorem isSublist_iff_sublist [DecidableEq α] {l₁ l₂ : List α} : l₁.isSublist l₂ ↔ l₁ <+ l₂ := by
+  cases l₁ <;> cases l₂ <;> simp [isSublist]
+  case cons.cons hd₁ tl₁ hd₂ tl₂ =>
+    if h_eq : hd₁ = hd₂ then
+      simp [h_eq, cons_sublist_cons, isSublist_iff_sublist]
+    else
+      simp only [h_eq]
+      exact ⟨
+        by
+          intro h_sub
+          apply Sublist.cons
+          exact isSublist_iff_sublist.mp h_sub,
+        by
+          intro h_sub
+          cases h_sub
+          case cons h_sub =>
+            exact isSublist_iff_sublist.mpr h_sub
+          case cons₂ =>
+            contradiction⟩
+
+instance instDecidableSublist [DecidableEq α] : DecidableRel (@Sublist α) := fun l₁ l₂ => by
+  rw [← isSublist_iff_sublist]
+  if h : l₁.isSublist l₂ then
+    exact isTrue h
+  else
+    exact isFalse h
```

Comment:
```suggestion
  decidable_of_iff isSublist_iff_sublist
```
untested, but something along these lines should work

Reply:
fixed in [fa204f1](https://github.com/leanprover/std4/pull/229/commits/fa204f15580b15e1a854e5618b47350d1ec118bf)